lib/vdsm/virt/vm.py

```diff
@@ -4443,6 +4443,13 @@ def diskReplicateFinish(self, srcDisk, dstDisk):
                            "srcDisk: %r, job: %r)",
                            drive.name, srcDisk, blkJobInfo)
 
+            # best-effort attempt to tear down the replica
+            try:
+                self.cif.teardownVolumePath(drive.diskReplicate)
+            except Exception:
```

Comment:
As this is a broad exception, maybe it would be useful for debugging later to catch it with `as e` and print the error cause? Or would it be too verbose?

Reply:
I never seen it failing but I think it's a good idea, it's not supposed to happen frequently - will do

Reply:
done